pom.xml

```diff
@@ -133,7 +133,8 @@
         
         <prometheus-simpleclient.version>0.11.0</prometheus-simpleclient.version>
         <prometheus-jmx.version>0.16.1</prometheus-jmx.version>
-        <opentelemetry.version>1.31.0</opentelemetry.version>
+        <opentelemetry.version>1.41.0</opentelemetry.version>
+        <opentelemetry-semconv.version>1.27.0-alpha</opentelemetry-semconv.version>
```

Comment:
Is the alpha version stable?

Reply:
- This is a requirement from https://opentelemetry.io/docs/languages/java/exporters/#otlp-dependencies and https://opentelemetry.io/docs/specs/semconv/ . The `Semantic Conventions` define a common set of (semantic) attributes which provide meaning to data when collecting, producing and consuming it. The `Semantic Conventions` specify among other things span names and kind, metric instruments and units as well as attribute names, types, meaning and valid values.
- The only reason it's in `alpha` is that the standards are converging with the `Elastic Common Schema`, and the convergence is not yet complete. `Elastic Common Schema` is another widely used common schema for handling observability and security data. See https://github.com/open-telemetry/oteps/blob/main/text/0199-support-elastic-common-schema-in-opentelemetry.md and https://www.elastic.co/blog/ecs-elastic-common-schema-otel-opentelemetry-faq/ . This shouldn't affect usage in theory.